Source/WebCore/bindings/js/ScriptController.h

```diff
@@ -130,8 +130,9 @@ class ScriptController final : public CanMakeWeakPtr<ScriptController>, public C
     TextPosition eventHandlerPosition() const;
 
     void setEvalEnabled(bool, const String& errorMessage = String());
+    void setTrustedEvalEnabled(bool);
     void setWebAssemblyEnabled(bool, const String& errorMessage = String());
-    void setRequiresTrustedTypes(bool);
+    void setRequiresTrustedTypes(bool, bool);
```

Comment:
Same problem here, but it’s even worse because this one we can’t even understand in the header here. We need names for these two `bool` arguments, but really we need a different clearer idiom.